ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/JavaParserExampleTest.java

```diff
@@ -0,0 +1,31 @@
+package edu.usb.argos.ASTProcessor;
+
+import org.junit.jupiter.api.Test;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class JavaParserExampleTest {
```

Comment:
Considering that you want to prove that the `JavaParser` dependency is working, you could increase it or put it inside a directory called `dependencyTests` to have more clarity of the names of the unit tests the change would be:
- `"ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/JavaParserExampleTest.java"`
to
- `"ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/dependencyTests/JavaParserExampleTest.java"`

Reply:
It was corrected

Reply:
According to what was discussed in class with the trainer, the conclusion was to eliminate the tests since the pull requests will already comply with the tests that are needed and this will only have the initialization of the javaparser library.